api-reference/_hidden/dxActionSheetItem/stylingMode.md

```diff
@@ -6,7 +6,12 @@ default: 'outlined'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies which style to apply to the button that represents an action sheet item.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following styles are available: 
+
+![ActionSheet Item Styling Modes](/images/UiWidgets/actionSheetItem_stylingMode.png)
+
+#####See Also#####
+- [type](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#type)
```

Comment:
Backlink from `type` to `stylingMode` maybe?

Reply:
Good point. Will add the same change to other members.